plugins/chat/db/post_migrate/20231227160005_make_type_on_chat_mentions_non_nullable.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class MakeTypeOnChatMentionsNonNullable < ActiveRecord::Migration[7.0]
+  def up
+    change_column :chat_mentions, :type, :string, null: false
+  end
+
+  def down
+    change_column :chat_mentions, :type, :string, null: true
+  end
```

Comment:
Should we use `change_column_null` instead?

Reply:
Fixed.